recipes/mqc/build.sh

```diff
@@ -0,0 +1 @@
+#! /bin/bash
```

Comment:
This file is empty, it should install your package.
So depending on your installation mechanism you need to add here: `python setup.py install ...` or simply copy your script to $PREFIX.